service/search/es_sync.go

```diff
@@ -243,42 +197,117 @@ func (e *esEngine) sync() error {
 		}
 		lastDelete = messages[len(messages)-1].DeletedAt.Time
 
-		bulkIndexer, _ := esutil.NewBulkIndexer(esutil.BulkIndexerConfig{
-			Client: e.client,
-			Index:  getIndexName(esMessageIndex),
-		})
+		err = syncDeletedMessages(e, messages, lastDelete, lastSynced)
+		if err != nil {
+			return err
+		}
+
+		if !more {
+			break
+		}
+	}
+
+	return nil
+}
+
+func syncNewMessages(e *esEngine, messages []*model.Message, lastInsert time.Time, lastSynced time.Time, userCache userCache) (err error) {
+	bulkIndexer, _ := esutil.NewBulkIndexer(esutil.BulkIndexerConfig{
+		Client: e.client,
+		Index:  getIndexName(esMessageIndex),
+	})
```

Comment:
これとsyncDeletedMessagesでも捨ててるのってerrorですか？
ならちゃんとhandlingしても良いと思う
```suggestion
	bulkIndexer, err := esutil.NewBulkIndexer(esutil.BulkIndexerConfig{
		Client: e.client,
		Index:  getIndexName(esMessageIndex),
	})
	if err != nil {
		return err
	}
```

Reply:
errorです。確かにハンドリングした方が良さそうなので、します。